config/jobs/kubernetes-sigs/cluster-api-provider-cloudstack/capi-provider-cloudstack-presumbit.yaml

```diff
@@ -0,0 +1,125 @@
+presets:
+- labels:
+    preset-image-build: "true"
+  env:
+  - name: AWS_SDK_LOAD_CONFIG
+    value: "true"
+  - name: GOPROXY
+  volumeMounts:
+  - name: docker-registry-config
+    readOnly: false
+    mountPath: /root/.docker
+  - name: run-buildkit
+    readOnly: false
+    mountPath: /run/buildkit
+  - name: entrypoint
+    readOnly: false
+    mountPath: /script
+  - name: status
+    readOnly: false
+    mountPath: /status
+  volumes:
+  # The files in this repo update the respective configmaps
+  # via the config-updater plugin in plugins.yaml for hook
+  - name: docker-registry-config
+    configMap:
+      name: build-setup
+      items:
+      - key: docker-ecr-config.json
+        path: config.json
+  - name: entrypoint
+    configMap:
+      name: buildkitd-entrypoint
+      items:
+      - key: buildkitd-entrypoint.sh
+        path: entrypoint.sh
+  - name: run-buildkit
+    emptyDir: {}
+  - name: status
+    emptyDir: {}
+- env:
+  - name: AWS_ROLE_SESSION_NAME
+    valueFrom:
+      fieldRef:
+        fieldPath: metadata.name
+
+
+presubmits:
+  kubernetes-sigs/cluster-api-provider-cloudstack:
+    - name: "capi-provider-cloudstack-presubmit-build"
```

Comment:
q: by a quick comparison with CAPI pre submit jobs it is missing the target branch. is it intentional?

Reply:
Per [presumbit spec's](https://pkg.go.dev/k8s.io/test-infra/prow/config#Presubmit) [Brancher spec](https://pkg.go.dev/k8s.io/test-infra/prow/config#Brancher)
```// Only run against these branches. Default is all branches.
	Branches [][string](https://pkg.go.dev/builtin#string) `json:"branches,omitempty"```